site/scripts/constants.js

```diff
@@ -33,13 +33,31 @@ export default {
   mangaModal: document.querySelector('#manga-modal'),
   settingsModal: document.querySelector('#settings-modal'),
   // Manga Details Modal
+  trackingImages: [
+    { src: 'img/trackers/ic_tracker_mal.webp', visible: false, syncId: 1, redirect: true },
+    { src: 'img/trackers/ic_tracker_anilist.webp', visible: false, syncId: 2, redirect: true },
+    { src: 'img/trackers/ic_tracker_kitsu.webp', visible: false, syncId: 3, redirect: true },
+    { src: 'img/trackers/ic_tracker_shikimori.webp', visible: false, syncId: 4, redirect: true },
+    { src: 'img/trackers/ic_tracker_bangumi.webp', visible: false, syncId: 5, redirect: true },
+    { src: 'img/trackers/ic_tracker_komga.webp', visible: false, syncId: 6, redirect: true },
+    {
+      src: 'img/trackers/ic_tracker_manga_updates.webp',
+      visible: false,
+      syncId: 7,
+      redirect: true,
+    },
+    { src: 'img/trackers/ic_tracker_kavita.webp', visible: false, syncId: 8, redirect: true },
+    { src: 'img/trackers/ic_tracker_suwayomi.webp', visible: false, syncId: 9, redirect: true },
+    { src: 'img/trackers/ic_tracker_mdlist.webp', visible: false, syncId: 60, redirect: true },
+  ],
```

Comment:
Can trim out the other fields 
```suggestion
  trackingImages: {
    1: 'img/trackers/ic_tracker_mal.webp',
    2: 'img/trackers/ic_tracker_anilist.webp',
    3: 'img/trackers/ic_tracker_kitsu.webp',
    4: 'img/trackers/ic_tracker_shikimori.webp',
    5: 'img/trackers/ic_tracker_bangumi.webp',
    6: 'img/trackers/ic_tracker_komga.webp',
    7: 'img/trackers/ic_tracker_manga_updates.webp',
    8: 'img/trackers/ic_tracker_kavita.webp',
    9: 'img/trackers/ic_tracker_suwayomi.webp',
    60: 'img/trackers/ic_tracker_mdlist.webp',
  },
```